sslh-main.c

```diff
@@ -287,6 +287,16 @@ int main(int argc, char *argv[], char* envp[])
 
    if (!cfg.foreground) {
        if (fork() > 0) exit(0); /* Detach */
+       // close stdin, stderr, stdout
+       int newfd;
+       if (newfd = open("/dev/null", O_RDWR)) {
+         dup2 (newfd, STDIN_FILENO);
```

Comment:
I am curious of why you're doing this. Intuitively I would have simply 
```
close(fileno(stdin));
close(fileno(stdout));
close(fileno(stderr));
```
(which is how we do it to close stderr when using inetd)

Reply:
I remembered this construct from Stevens Network Programming, but I have to reread to give more arguments.
Just copied from some older simple programs, I wrote decades ago

Reply:
But I bet, the other way is also working.